util/linkpreview/cache.go

```diff
@@ -32,11 +32,11 @@ func (c *cache) Name() string {
 	return CName
 }
 
-func (c *cache) Fetch(ctx context.Context, url string) (lp model.LinkPreview, body []byte, err error) {
+func (c *cache) Fetch(ctx context.Context, url string) (lp model.LinkPreview, body []byte, b bool, err error) {
```

Comment:
**b** should be also renamed to **isFile**, isn't it?

Reply:
Fixed